AppCenterDistribute/AppCenterDistributeTests/Util/MSMockReachabilityUtil.m

```diff
@@ -0,0 +1,35 @@
+#import "MSMockReachabilityUtil.h"
+#import "MSTestFrameworks.h"
+
+static NSString *kMSNetworkReachabilityChangedNotificationName = @"kMSNetworkReachabilityChangedNotification";
+
+@interface MSMockReachabilityUtil ()
+
+@property(nonatomic) id mockReachabilityUtil;
+
+@end
+
+@implementation MSMockReachabilityUtil
+
+@synthesize currentNetworkStatus = _currentNetworkStatus;
+
+- (void)mockMSReachability {
```

Comment:
```suggestion
+ (id)mockReachability {
```

Reply:
1. static: no need to create an instance of util class
2. return type: Just return mock object here

Reply:
Good idea also if we have `setCurrentNetworkStatus`, `startMocking` and `stopMocking` as class methods we won't really need to manipulate instances in tests.

Reply:
I implement the class method setCurrentNetworkStatus and startMocking, and startMocking will return a mock object for ms_reachability. But I think we don't need to implement the stopMocking method, because we can directly use the mock object's stopMocking method int the test. (the object is returning from startMocking method)